basic_features/basic_save_game_info.py

```diff
@@ -33,7 +33,7 @@ def allFiles(self) -> list[str]:
 
 
 class BasicGameSaveGameInfoWidget(mobase.ISaveGameInfoWidget):
-    def __init__(self, parent: QWidget, get_preview: Callable[[Path], Path | None]):
+    def __init__(self, parent: QWidget, get_preview: Optional[Callable[[Path], Optional[QPixmap | Path | str]]]):
```

Comment:
`Optional` should not be used anymore, you can use `| None` when needed. I don't think making `get_preview` optional is needed, simply default it to a `lambda p: None`. I thought this was already done in an existing PR 🤔